conf/modules.config

```diff
@@ -200,6 +210,56 @@ process {
     }
 }
 
+//
+// Sentieon alignment options
+//
+
+process{
+    withName: '.*ALIGN_SENTIEON:.*' {
+        ext.when = params.aligner.equals("sentieon")
+    }
+
+    withName: '.*ALIGN:ALIGN_SENTIEON:SENTIEON_BWAMEM' {
+        ext.args = { "-M -K 10000000 -R ${meta.read_group} " }
+        ext.prefix = { "${meta.id}.sorted" }
+        publishDir = [
+            path: { "${params.outdir}/sentieon_align" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
+        ]
+    }
+
+    withName: '.*ALIGN:ALIGN_SENTIEON:SENTIEON_DATAMETRICS' {
+        publishDir = [
+            path: { "${params.outdir}/sentieon_bam_metrics" },
+            mode: params.publish_dir_mode,
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
+        ]
+    }
+
+    withName: '.*ALIGN:ALIGN_SENTIEON:SENTIEON_LOCUSCOLLECTOR' {
+        publishDir = [
+            path: { "${params.outdir}/sentieon_dedup" },
+            mode: params.publish_dir_mode,
+        ]
+    }
+
+    withName: '.*ALIGN:ALIGN_SENTIEON:SENTIEON_DEDUP' {
+        publishDir = [
```

Comment:
Can we add [--rmdup] option as an ext.args here? 
ext.args ={$parms.rmdup ? " --rmdup " : '' }